VERSION

```diff
@@ -1 +1 @@
-0.9.0
+0.9.1
```

Comment:
I strongly suggest not to mix unrelated logic to a single PR.

Reply:
Done